### DISCUSSION TITLE
Near-term vetting: Hydro

### DISCUSSION THREAD

**Discussion by pweigmann** [pik-piam/mrremind#541] [Near-term validation]
**Variable: `Capacity|Electricity|Hydro`**

**Source: IEA-HSMR**

* country data
* available data: 
    * "operational": currently operational capacities 
    * "expected": most likely scenario for 2030 capacities 
    * "accelerated": scenario with accelerated capacity growth until 2030
* pumped storage is separately reported for 2020 and 2030, but only for the "expected" scenario
    * for the calculations below, pumped storage is excluded from the "operational", "expected" and "accelerated" and handled by its own for greater flexibility in threshold construction

Assumptions + data preparation:

* slow technology: potential new power plants that are not yet under construction won't be operational in 2030
* all power plants under construction will be finished in 2030 (only `max` thresholds)
* no decommissioning
* `min` thresholds without pumped storage, `max` thresholds with pumped storage as different models might include or exclude it in their reporting

Threshold definitions:

**2020**
* global  `min_red`: -10% ("operational") ; regional  `min_red`: -40% ("operational")
* global `min_yel`:  -5% ("operational")   ; regional  `min_yel`: -20% ("operational")
* global `max_yel`: +5% ("operational" + "pumped storage")  ; regional  `max_yel`: +20% ("operational" + "pumped storage")
* global `max_red`: +10% ("operational" + "pumped storage") ; regional  `max_red`: +40% ("operational" + "pumped storage")

**2030**
* global  `min_red`: -10% ("operational") 
* global `min_yel`:  -5% ("expected")     ; regional  `min_yel`: -40% ("operational") 
* global `max_yel`: +5% ("accelerated" + "pumped storage")   ; regional  `max_yel`: +40% ("accelerated" * 1.3 + "pumped storage") 
* global `max_red`: +10%  ("accelerated" * 1.3 + "pumped storage") 

Data Source:
https://www.iea.org/data-and-statistics/data-tools/hydropower-data-explorer

**Comment by EHooijschuur** [comment]
The data that I received a while ago, has a remarkable drop in 2023. Do we know where that comes from? Maybe data from a specific country is not available that year?

Additionally, there are a few sources that show lower historical capacities: IRENA, WRI, Ember, Platts. Is there a specific reason you chose to use IEA? Maybe, for a lower boundary, it would make sense to choose sources that show lower capacities, or choose a higher uncertainty range because numbers differ.

**Reply by pweigmann** [reply]
In general, we chose IEA for its near-term outlook, offering an "expected" and "accelerated" scenario for hydro capacities.

I am not certain, which data it is that contains the drop in 2023 - in this case we use 2020 as a historical comparison year and then add thresholds also for 2030. Regarding the 2020 data, I found the following global capacities:

- IEA: 1168 GW + 160 GW pumped storage
- EMBER: 1213 GW
- IRENA: 1156 GW

I didn't find the numbers for WRI and Platts in my quick search, but generally I think the IEA shouldn't be too far off. Maybe I am looking at different numbers than you?
